frontend/octavio/package.json

```
@@ -0,0 +1,61 @@
+{
+  "name": "ictsc_sachiko_3",
+  "version": "0.1.0",
+  "private": true,
+  "scripts": {
+    "dev": "next dev",
+    "build": "next build",
+    "start": "next start",
+    "test": "vitest --run --coverage",
+    "test:e2e": "playwright test",
+    "test:e2e-ui": "playwright test --ui",
+    "lint": "next lint",
+    "format": "prettier --check --ignore-path .gitignore .",
+    "format:fix": "prettier --write --ignore-path .gitignore .",
+    "font": "docker build -f Dockerfile.font -t font-builder . && docker run -v $(pwd):/host -it font-builder sh -c \"cp /font/Variable/OTF/Subset/NotoSansJP-VF.woff2 /host/pages/NotoSansJP-VF.woff2 && exit\""
```

今回のPRと直接関係は無いですが、以下のように記述すると、マルチステージのDockerfileで指定したステージのみイメージとして出力できるので、これに置き換えて`Dockerfile.font`削除して欲しいです
```suggestion
    "font": "docker build --target font-builder -t font-builder . && docker run -v $(pwd):/host -it font-builder sh -c \"cp /font/Variable/OTF/Subset/NotoSansJP-VF.woff2 /host/pages/NotoSansJP-VF.woff2 && exit\""
```

---

こうすればよかったのか・・感謝です🙏